src/app/Dashboard/operations/logPoints.data.js

```diff
@@ -0,0 +1,88 @@
+/* eslint-disable max-len */
+import { put, takeEvery, call } from 'redux-saga/effects';
+
+import actions from './actions';
+import types from './types';
+
+const fetchCategoriesUrl = 'https://private-anon-6062be0d7e-andelasocietiesapi.apiary-mock.com/api/v1/activity-types';
```

Comment:
Let's use constants defined in `config` file.

Reply:
@Chris-Maina I have removed it and used the constants in the config file